app/uk/gov/hmrc/tradergoodsprofiles/services/RouterService.scala

```diff
@@ -101,7 +154,7 @@ class RouterServiceImpl @Inject() (
             )
             Left(
               ServerErrorResponse(
-                dateTimeService.timestamp,
+                uuidService.uuid,
```

Comment:
correlationId ID here has not meaning and it will only create confusion for the user or user support team as this is used to track the request downstream (to EIS), but in here we haven\t set the request to EIS yet. Also this will conflict the correlationID from EIS